storage/src/tests/distributor/simplemaintenancescannertest.cpp

```diff
@@ -80,7 +94,27 @@ SimpleMaintenanceScannerTest::testPrioritizeSingleBucket()
     addBucketToDb(1);
     std::string expected("PrioritizedBucket(Bucket(BucketSpace(0x0000000000000000), BucketId(0x4000000000000001)), pri VERY_HIGH)\n");
 
-    CPPUNIT_ASSERT(!_scanner->scanNext().isDone());
+    auto scanResult = _scanner->scanNext();
+    CPPUNIT_ASSERT(!scanResult.isDone());
+    CPPUNIT_ASSERT_EQUAL(scanResult.getBucketSpace().getId(), makeBucketSpace().getId());
```

Comment:
1st argument to macro is the expected value, so flipping the arguments around will ensure that correct expected/actual values are printed.